### PR TITLE
feat(components/popovers): keyboard navigation for popovers with interactable content

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[AB#2836531](https://dev.azure.com/blackbaud/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2836531)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2294/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/errors-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/forms-storybook/)
- [help-inline-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/help-inline-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/lookup-storybook/)
- [navbar-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/navbar-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/phone-field-storybook/)
- [popovers-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/popovers-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2294/storybooks/theme-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2294/code-examples/)
- [dep-graph](https://blackbaud.github.io/skyux-pr-preview/2294/dep-graph/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2294/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2294/playground/)

File: libs/components/popovers/src/lib/modules/popover/popover-content.component.ts [hunk 1]
```diff
@@ -133,6 +134,8 @@ export class SkyPopoverContentComponent implements OnInit, OnDestroy {
   #context: SkyPopoverContext;
   #themeSvc: SkyThemeService | undefined;
 
+  #skyAdapterService = inject(SkyCoreAdapterService);
```
Comment (Blackbaud-SteveBrush) [inline libs/components/popovers/src/lib/modules/popover/popover-content.component.ts]: Let's follow convention and set this to:
```
readonly #adapterSvc = inject(SkyCoreAdapterService);
```

File: libs/components/popovers/src/lib/modules/popover/popover.component.ts [hunk 2]
```diff
@@ -231,6 +231,14 @@ export class SkyPopoverComponent implements OnDestroy {
     this.#isMarkedForCloseOnMouseLeave = true;
   }
 
+  /**
+   * Whether the popover body has interactable content
```
Comment (Blackbaud-SteveBrush) [inline libs/components/popovers/src/lib/modules/popover/popover.component.ts]: This is a component, so we don't need JSDocs (or `@internal`) for methods/properties (we only do that for services).

Comment (Blackbaud-SteveBrush) [reply libs/components/popovers/src/lib/modules/popover/popover.component.ts]: Also, the method name is self-explanatory, so we can just remove the comments. Good forward thinking, though. 👍

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Pull request merged**